src/queue/processors/inbox.ts

```diff
@@ -144,5 +154,6 @@ export default async (job: Bull.Job<InboxJobData>): Promise<string> => {
 
 	// アクティビティを処理
 	await perform(authUser.user, activity);
+	await perform(authUser.user, activity);
```

Comment:
Is there any significance to the duplication here?

Reply:
don't think so, i wonder if it was added when i merged the updates